build/pre-migration-test-docker.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+###############################################################################
+# (C) Copyright IBM Corp. 2016, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+set -ex
+
+echo "Preparing environment for fhir-server integration tests..."
+if [[ -z "${WORKSPACE}" ]]; then
+    echo "ERROR: WORKSPACE environment variable not set!"
+    exit 2
+fi
+
+# Set up installers and config files where docker processing can see them
+cd ${WORKSPACE}/fhir-install/docker
+./copy-dependencies-db2-migration.sh
```

Comment:
this script is much like `pre-integration-test-docker.sh` except it calls a modified copy-dependencies script that downloads a previous schema cli jar instead of taking the newly built one from fhir-persistence-schema/target